### DISCUSSION TITLE
Feature Request: Show advanced graphs and statistics.

### DISCUSSION THREAD

**Discussion by upuldi** [babybuddy/babybuddy#451] [Ideas]
Things like 

1. Feeding change over time.
2. Napping duration changes over time. 
3. etc

**Comment by myxor** [comment]
I started implementing reports for feeding intervals over time and diaper changes intervals over time here: https://github.com/babybuddy/babybuddy/pull/444

**Comment by cdubz** [comment]
I've actually been thinking recently about _removing_ graphs and letting people use third-party tools and Baby Buddy's API for greater flexibility. I know some users have used e.g. [Grafana](https://grafana.com/) for this.

That said until I have more firm plans around that I'm happy to review and consider PRs for new graphs! I'll watch out for your PR to be review ready, @myxor.

I'm going to convert this to a discussion for further... discussion and folks can add issues and open PRs for specific graphs.
